beets/util/__init__.py

```diff
@@ -769,7 +769,7 @@ def shlex_split(s):
         raise TypeError('shlex_split called with non-string')
 
 
-def interactive_open(targets, command):
+def interactive_open(targets, command, take_over=True):
```

Comment:
Perhaps a more intuitive name for the parameter would be `fork` (i.e., the default is `fork=False`).